charts/hub/portworx/templates/aws-prereqs/px-aws-prereqs-cm.yaml

```diff
@@ -0,0 +1,207 @@
+{{- if eq $.Values.global.clusterPlatform "AWS" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+  name: px-aws-prereqs-configmap
+  namespace: {{ .Values.namespace }}
+data:
+  px_aws_pre-reqs.sh: |	
+    #!/bin/bash
+
+    export AWS_CONFIG_FILE=/pattern-home/credentials/.aws/config
+    export AWS_SHARED_CREDENTIALS_FILE=/pattern-home/credentials/.aws/credentials
+
+    mkdir /pattern-home/credentials/.aws
+
+    # Ensure we can access the secret for AWS credentials
+    oc get secret aws-creds -n kube-system -o json
+    if [ $? != 0 ];
+    then
+      echo "Cannot access aws-creds secret in kube-system namespace, please check rbac for portworx-prereq-sa serviceAccount."
+    else
+      echo "Found AWS credential secret, building credentials file."
+      AWS_ACCESS_KEY=$(curl -sSk -H "Authorization: Bearer $(cat /run/secrets/kubernetes.io/serviceaccount/token)" https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_PORT_443_TCP_PORT/api/v1/namespaces/kube-system/secrets/aws-creds | jq -r '.data.aws_access_key_id' | base64 -d)

```

Comment:
Maybe use `oc get -n kube-system secret/aws-creds -o go-template='{{ index .data "aws_access_key_id" | base64decode }}'`. Since you already use oc above, this is a bit simpler/easier to read, I think